scylla/src/utils/pretty.rs

```diff
@@ -51,7 +51,7 @@ where
             CqlValue::Inet(i) => write!(f, "'{}'", i)?,
             CqlValue::SmallInt(si) => write!(f, "{}", si)?,
             CqlValue::TinyInt(ti) => write!(f, "{}", ti)?,
-            CqlValue::Varint(vi) => write!(f, "{}", vi)?,
+            CqlValue::Varint(vi) => write!(f, "0x{:x}", HexBytes(vi.as_signed_bytes_be_slice()))?,
```

Comment:
Suggestion: change the format string to `"blobAsVarint(0x{:x})"` - it will be actually be a valid cql expression.